app/models/event.py

```diff
@@ -43,7 +43,7 @@ class Event(SoftDeletionModel):
     starts_at = db.Column(db.DateTime(timezone=True), nullable=False)
     ends_at = db.Column(db.DateTime(timezone=True), nullable=False)
     timezone = db.Column(db.String, nullable=False, default="UTC")
-    is_event_online = db.Column(db.Boolean)
+    is_event_online = db.Column(db.Boolean, default=False)
```

Comment:
This will require you to run the migrations. Right ?

Reply:
Migrations add default values as specified in models only when the attr is required. 
Here it's not mentioned that it's value is required in the model or schema.